packages/generator-rest/test/poc/generated/PriceCalculation/request-builder.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved.
+ *
+ * This is a generated file powered by the SAP Cloud SDK for JavaScript.
+ */
+import { Destination, DestinationNameAndJwt, RestRequestBuilder, RestRequestConfig } from '@sap-cloud-sdk/core';
+import { AxiosRequestConfig } from 'axios';
+import { PriceCalculationApi, PriceCalculate } from './api';
+
+export class PriceCalculationApiRequestBuilder {
```

Comment:
This is the generated request builder.

Reply:
For discussion: @FrankEssenberger @jjtang1985 Do you think the name should be `-RequestBuilder` or rather `-Api`? Although structurally it is very similar to our other request builders, here you don't really build construct a request (with a fluent api) as in the other one.

Reply:
You are right, the reasons why I use `RequestBuilder` are:
- the naming conflict between this and the one generated by openapi
  - if we decide to hide the generated open api client, then we can only export this
  - if we want to export both, we have to make a second name
- the similar structure.

Reply:
Discussed with Frank, I still think the vanilla version of the open api client can work as some work around in some cases. Therefore, choosing a new name might make sense. The name of the class still needs to be decided though.